app/views/works/index.html.erb

```diff
@@ -0,0 +1,122 @@
+<div class="alert"> TODO: ADD LINKS TO UPVOTE AND ACCURATE VOTE COUNT </div>
+<%# remove the above code before final submission%>
+
+<h2> List of Works </h2>
+<section class="media-table">
+  <h4> Movies </h4>
+  <% if !Work.list("movie").empty? %>
+    <table class="table">
+      <thead>
+        <tr>
+          <th width="80"> Votes </th>
+          <th> Title </th>
+          <th> Created By </th>
+          <th width="100"> Published </th>
+          <th width="100"> Upvote </th>
+        </tr>
+      </thead>
+      <% Work.list("movie").each do |movie|%>
+        <tr>
+          <td> 
+            <%= movie.id %>
+          </td>
+          <td> 
+            <%= link_to movie.title, work_path(movie.id) %>
+          </td>
+          <td> 
+            <%= movie.creator %>
+          </td>
+          <td> 
+            <%= movie.publication_year %>
+          </td>
+          <td> 
+            <%= link_to "Upvote", work_upvotes_path(movie.id), method: :post %>
+          </td>
+        </tr>
+      <% end %>
+    </table>
+  <% else %>
+    No media available
+  <% end %>
+</section>
+
+<section class="media-table">
+  <h4> Albums </h4>
+  <% if !Work.list("album").empty? %>
```

Comment:
You have this code repeated 3 times. Use a view partial or view helper to dry this up!